rescue/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+// use std::marker::PhantomData;
+
+// use p3_field::{Field, PrimeField, PrimeField64};
+// use p3_symmetric::permutation::{MDSPermutation, CryptographicPermutation, ArrayPermutation};
+
+mod inverse_sbox;
+mod mds_matrix_naive;
+mod rescue;
+mod util;
```

Comment:
Should we add `pub use rescue::*;` so `Rescue` is visible?

Reply:
Oh yes!